scripts/demo.sh

```diff
@@ -10,7 +10,7 @@
 # then run this script
 
 
-ALGO="--bft"
+ALGO="--real-pbft"
```

Comment:
Ahhh. Ok, rebase & squash this so we're not switching back and forth within this PR.